hadoop-cloud/pom.xml

```diff
@@ -267,6 +267,13 @@
               <groupId>com.google.guava</groupId>
               <artifactId>guava</artifactId>
             </exclusion>
+            <exclusion>
+              <!--
+              This is a code coverage library introduced by aliyun-java-sdk-core, only for testing
+              -->
+              <groupId>org.jacoco</groupId>
+              <artifactId>org.jacoco.agent</artifactId>
```

Comment:
OK yeah I think we can safely exclude it. Probably a mistake that it wasn't test-scope in the original artifact